packages/functions/e2e/functions.e2e.js

```diff
@@ -298,7 +298,7 @@ describe('functions()', () => {
         await functions.httpsCallable(fnName, { timeout: 1000 })({ testTimeout: 3000 });
         return Promise.reject(new Error('Did not throw an Error.'));
       } catch (error) {
-        error.message.should.containEql('DEADLINE_EXCEEDED');
+        error.message.should.containEql('UNAVAILABLE');
```

Comment:
Any specific reason you changed this rather than fixing the underlying issue I mentioned here:

https://github.com/invertase/react-native-firebase/pull/3534#issuecomment-618955152

Reply:
Ah, I interpreted that as a tweak to test logic because this code was approved a while back: https://github.com/invertase/react-native-firebase/pull/3327/files

Happy to change back if that's what you want?